Assets/PatchKit Patcher/Scripts/Background.cs

```diff
@@ -110,12 +111,25 @@ private void OnBannerDataUpdate(Data data)
         _logger.LogDebug("On patcher data update.");
         var bannerData = data.BannerData;
 
-        if (string.IsNullOrEmpty(bannerData.ImageUrl))
+        if (string.IsNullOrEmpty(bannerData.ImageUrl) && string.IsNullOrEmpty(bannerData.ModificationDate))
```

Comment:
I'm wondering if we could simplify this my merging this condition with condition on line 126.
1. In both cases default patcher should be removed.
2. Calling ClearCachedBanner() should have no effect when there's no cached banner, but it still should log an information that there's no cached banner.
3. I'm not sure about animations.

What do you think?